docs/Installation/System-Requirements.md

```diff
@@ -0,0 +1,29 @@
+title: System Requirements
+description: System requirements to sync and run Pantheon 
+<!--- END of page meta data -->
+
+# System Requirements 
+
+The system requirements for Pantheon depend on a number of factors: 
+
+* Network - are you connecting to a public Ethereum network (eg, mainnet) or a private network 
+* Network size if connecting to a private network 
```

Comment:
The network you connect to doesn't really matter, it's how big the world state is for that network, how many transactions are submitted to the network and how high the block gas limit is.

Reply:
Fixed